.env.sample

```diff
@@ -14,9 +14,9 @@ AUTH_API_KEY_FILE=/apikey/sample.csv
 
 TEST_DATA={"key_manager_url":"https://localhost:8080","health_key_manager_url":"https://localhost:8081","secret_stores":["hashicorp-secrets","akv-secrets","aws-secrets"],"key_stores":["hashicorp-keys"],"eth1_stores":["eth1-accounts"],"quorum_node_id":"quorum-node","besu_node_id":"besu-node"}
 
-## Postgres Configuration
-DB_TLS_SSLMODE=verify-ca
-DB_TLS_CERT=/certificates/client.crt
-DB_TLS_KEY=/certificates/client.key
-DB_TLS_CA=/ca/ca.crt
-DB_HOST=postgres-ssl
+## Postgres SSL
```

Comment:
this is no longer working?

Reply:
I commented for our dev envs, since I was running into the permission issue of the certs. IMO it is not very necessary to be running local dev with SSL PG if that brings complexity to our dev environments. I would rather validate the SSL only on our e2e once we get to run it using remote PG